peps/pep-0728.rst

```diff
@@ -384,14 +396,14 @@ For example::
 
     class MovieBase(TypedDict, extra_items=int | None):
         name: str
-    
-    class AdaptedMovie(MovieBase):  # Not OK. 'bool' is not assignable to 'int | None'
+
+    class AdaptedMovie(MovieBase):  # Not OK. 'bool' is required
```

Comment:
AdaptedMovie and MovieRequiredYear seem redundant with each other now. They're both examples of a required key.

Reply:
Right, removed the adaptation. I think the one with the required `int | None` key is more interesting because it shows that you can't add a required item even if the type matches the `extra` type exactly.